draft-ietf-tls-tls13.md

```diff
@@ -4643,11 +4672,6 @@ bad_certificate_status_response
   provided by the server via the "status_request" extension
   (see {{RFC6066}}).
 
-bad_certificate_hash_value
-: Sent by servers when a retrieved object does not have the correct hash
-  provided by the client via the "client_certificate_url" extension
-  (see {{RFC6066}}).
```

Comment:
These alerts are only usable for client_certificate_url, which we deemed to be silly and removed a while ago.